reports-scheduler/src/main/kotlin/org/opensearch/reportsscheduler/notifications/NotificationsActions.kt

```diff
@@ -89,7 +90,7 @@ internal object NotificationsActions {
             NotificationsPluginInterface.sendNotification(
                 client,
                 EventSource(delivery.title, referenceId, FEATURE_REPORTS, SeverityType.INFO),
-                ChannelMessage(delivery.textDescription, delivery.htmlDescription, null),
+                ChannelMessage("${delivery.textDescription}\nGet your report at $reportLink", delivery.htmlDescription, null),
```

Comment:
should htmlDescription also be modified? i remember previously testing email channels it only receives htmlDescription

Reply:
Correct me if I am wrong. Now the htmlDescription is passed from reporting UI plugin, adding embedded html and stuff are still in the UI plugin. But I think we should move some logic to the backend 1) logic of converting message text to html 2) adding Opensearch email template.

After that, the report link will be added to the logic above when building the html. I say that's a cleaner solution

Reply:
Meging this one, I'll create another issue for email and send another PR